doc/changes/latest.inc

```diff
@@ -115,6 +115,8 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug with :func:`mne.Epochs.plot_image` where the ``x_label`` was different depending on the evoked parameter (:gh:`9115` **by new contributor** |Matteo Anelli|_)
```

Comment:
It's a trick to get a bold name link in RST. You need to add a `replace::` line at the top to match (see other examples)

Reply:
okay thanks, that makes sense

Reply:
Thank you, I'll do it.